.eslintrc.js

```diff
@@ -128,6 +128,7 @@ module.exports = {
     "func-style": 0,
     "indent": [2, 2, { "SwitchCase": 1 }],
     "key-spacing": [2, { "beforeColon": false, "afterColon": true }],
+    "keyword-spacing": [2, { "before": true, "after": true }],
```

Comment:
"space-after-keywords" and "space-return-throw-case" are deprecated and rolled into this one rule now.